openedx/core/lib/api/tests/mixins.py

```diff
@@ -0,0 +1,56 @@
+"""
+Mixins for API Authentication tests.
+"""
+from time import time
+
+from django.conf import settings
+from django.test import TestCase
+import jwt
+
+
+JWT_AUTH = 'JWT_AUTH'
+
+
+class JwtMixin(TestCase):
+    """ Mixin with JWT-related helper functions. """
+    def setUp(self):
```

Comment:
i don't think you should define `setUp` in a mixin (I mentioned this in an edit to my original comment)